pkg/istio/authorizationpolicy/controller.go

```diff
@@ -117,6 +118,15 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	go c.authzpolicyIndexInformer.Run(stopCh)
 	go c.resync(stopCh)
 
+	if c.configStoreCache.HasSynced() {
```

Comment:
Can we actually add a wait for cache sync here instead of an if statement like we do here: https://github.com/yahoo/k8s-athenz-istio-auth/blob/master/pkg/controller/controller.go#L216-L218
We need to wait until it has fully synced before processing anything. This if statement will just check it once.